### PR TITLE
Re-design integration test for Accuracy trace

### PR DESCRIPTION
**Pull request by geez0219**

### CODE REVIEW

File: test/PR_test/integration_test/trace/metric/test_accuracy.py [hunk 1]
```diff
@@ -14,70 +14,80 @@
...
-    def test_on_batch_end(self):
-        self.accuracy.on_batch_end(data=self.data)
-        with self.subTest('Check correct values'):
-            self.assertEqual(self.accuracy.correct, 1)
-        with self.subTest('Check total values'):
-            self.assertEqual(self.accuracy.total, 3)
+    def test_tf_binary_class(self):
+        with self.subTest("from_logit=False"):
+            trace = Accuracy(true_key="label", pred_key="pred", output_name=self.acc_key, from_logits=False)
+            batch = {"label": tf.constant([[1, 0], [0, 1]])}
```
Comment (vbvg2008) [inline test/PR_test/integration_test/trace/metric/test_accuracy.py:48]: if it is binary classification, then label won't be in shape [X, 2].

**Review by TortoiseHam — APPROVED** [review]
LG
